addons/payment_stripe/controllers/main.py

```diff
@@ -171,3 +169,40 @@ def _verify_webhook_signature(self, webhook_secret):
             return False
 
         return True
+
+    # -------------------------------------------
+    # Stripe Connect Onboarding
+    # -------------------------------------------
+    @http.route('/payment/stripe/onboarding/return/<int:acquirer_id>', type='http', auth='public', csrf=False)
```

Comment:
Why does it need to be auth='public'?

Reply:
In order to work even if the session has been closed. So the user only has to re-login to access the right page. Don't you think it's a good idea ?

Reply:
@mart-e please correct me if I'm wrong but if their session expires, Odoo would just create a new one for them. `auth=` refers to whether the associated `res.user` must be internal or not.

Reply:
No `auth=user` -> authenticated with a res.users (so portal included)